### PR TITLE
homework-4 PR

### PR DESCRIPTION
**Pull request by najahizquierdo**
Need to figure out why my repository isnt allowing me to push from the terminal.

### CODE REVIEW

**Comment by najahizquierdo** [comment]
@dafrenchyman @brian004900

File: homework4/assignment4.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+# check if response is cont or bool
+def response_bool_or_cont(col):
+    response = col.target_names.tolist()
+    if response is bool or str:
+        return True
+    elif len(response) == 2:
+        return True
+    else:
+        return False
+
```
Comment (brian004900) [inline homework4/assignment4.py:25]: This def can determine if the response is con or cat.
The predictors can also be resolved this way.

File: homework4/assignment4.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    if response_bool_or_cont(data) is True:
+        response = "boolean"
+        print("This is a boolean.")
+    else:
+        response = "continuous"
+        print("This is continuous.")
+
+    # linear regression
+    if response == "continuous":
+        for a, col in enumerate(X.T):
```
Comment (brian004900) [inline homework4/assignment4.py]: p-value & t-score are for continuous predictors only.

File: homework4/assignment4.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+            )
+            # fig.show()    # Categorical
+    elif response == "boolean":
+        # logistic regression
+        for a, col in enumerate(X.T):
+            print(f"Running logistic regression for: {response} ")
+            col = X[:, a]
+            feature_name = data.feature_names[a]
+            predictor = statsmodels.api.add_constant(col)
+            logi_regression_model = statsmodels.api.Logit(
```
Comment (brian004900) [inline homework4/assignment4.py:184]: successfully did logistic regression.

**Pull request merged**